lib/charms/tempo_k8s/v1/charm_tracing.py

```diff
@@ -244,11 +266,10 @@ def wrap_init(self: CharmBase, framework: Framework, *args, **kwargs):
         server_cert: Optional[str] = (
             _get_server_cert(server_cert_getter, self, charm) if server_cert_getter else None
         )
-        credentials = ChannelCredentials(server_cert) if server_cert else None
-        insecure = None if credentials else True
 
+        # OTLP HTTP exporter expects a cert file path instead of certificate itself that OTLP GRPC did
```

Comment:
remove this comment. it's good to know for this review but it doesn't need to remain there forever